examples/quickstart.ipynb

```diff
@@ -4,7 +4,26 @@
    "cell_type": "markdown",
```

Comment:
<p>How about "using <code>if</code> statements.", instead of "using Python <code>if</code>s."?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/optuna/optuna/pull/804/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Thanks for the catch, I agree 100%. Fixed as suggested.</p>